data/sql/updates/pending_db_world/rev_1638737032167850775.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1638737032167850775');
+
+-- Makes quest Lazy Peon broadcast work work only from themselves and not the player (action_para2)
+-- Sets it up where the target_type is the player and only triggers with in 20 yards
+UPDATE `smart_scripts` SET `action_param2`=1, `target_type`=18, `target_param1`=20 WHERE  `comment`="Lazy Peon - On Script - Play Sound 6197";
```

Comment:
Use single quotes instead of doubles :D

Reply:
This is an unusual where clause, any particular reason why source_type & entryorguid where not used?

Reply:
@Nefertumm 
are you referring to the comment portion?

@Nyeriah 
It was to minimize the targeted lines dealing with the audio only. The lazy peons had a mess load of guid's and entries individually per the creature and not a blank cover all for a single entry type.